backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Mocks/Class1.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Altinn.AccessManagement.UI.Mocks
+{
+    public class Class1
```

Comment:
I assume this is an unintentionally added class

Reply:
Yup, very much unintentional 😅